sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -80,7 +80,21 @@ object Radar {
             events: Array<RadarEvent>? = null,
             user: RadarUser? = null
         )
+    }
+
+    /**
+     * ,,,
+     */
+    interface RadarFlushReplaysCallback {
 
+        /**
+         * ...
+         *
+         * @param[status] RadarStatus The request status.
+         */
+        fun onComplete(
+            status: RadarStatus
+        )
```

Comment:
Hmmm, thinking that we might want `events` on this too, given the event cache we're building out. We'd just hand it off the events event delegate.

Reply:
(for now, implementing as is in iOS)